src/index.js

```diff
@@ -1,2 +1 @@
-import { Listbox, ListboxButton, ListboxLabel, ListboxList, ListboxOption } from './Listbox'
-export { Listbox, ListboxButton, ListboxLabel, ListboxList, ListboxOption }
+export { Listbox, ListboxButton, ListboxLabel, ListboxList, ListboxOption } from './Listbox'
```

Comment:
This could be changed to
```suggestion
export * from './Listbox'
```
And this way you don't need to write all the imports

Reply:
True. I do find that this makes the public API less clear and it exposes more publicly than what I’d like (it removes the ability use exports for other purposes).

I’ll let Adam decide.